proseco/characteristics_guide.py

```diff
@@ -9,6 +9,12 @@
 # Minimum scaled guide count for thumbs_up
 min_guide_count = 4.0
 
+# Add this padding to region checked for bad pixels (in addition to dither)
+dither_pix_pad = 0.4
+
+# Minimum imposter mag (if dark current is 0 set explicitly to this value)
+min_imposter_mag = 20.0
```

Comment:
This is actually the *max* value of `imposter_mag`.  Otherwise this and the code that uses it are OK.

Reply:
Hah.  I should have used a variant on faint.

Reply:
How about "imposter_mag_for_zero_dark"?

Reply:
Fine.  Plan B would be to replace `pixmax` with `np.clip(pixmax, 1.0, None)` in the offending line, add a code comment, and be done with it.  I somehow feel like characteristics should be reserved for constants that have a discernible algorithmic impact.  But that is my style and not necessarily correct.  Your choice.  FYI, count rate of 1.0 is very nearly 20th mag.  :smile:.

Reply:
Sure.  I like that better.  And yes, I saw that count rate of 1 is basically 20th mag.  I considered just setting pixmax to 1.0 before starting the loop over the sums, but didn't want to just add 1.0 in general.  Your clip is better.